source/pages/uv-dose.ipynb

```diff
@@ -0,0 +1,146 @@
+{
```

Comment:
<p>Can we call it <code>VARIABLE_MAP</code> to be consistent with pvlib? :D</p><p> </p><p>Also, is there a reason that you are using hyphens and not underscores in the mapped variable names?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AssessingSolar/unofficial-psm3-userguide/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
> Also, is there a reason that you are using hyphens 

Just matching the PSM3 API, which uses hyphens for the two ghuv fields: https://developer.nrel.gov/docs/solar/nsrdb/psm3-download/